pkg/transform/spanner.go

```diff
@@ -94,13 +84,21 @@ func extractIP(b []uint8, size int) string {
 	return net.IP(b[:size]).String()
 }
 
-func (c *converter) convert(trace *ebpfcommon.HTTPRequestTrace) HTTPRequestSpan {
-	now := c.clock()
-	monoNow := c.monoClock()
-	startDelta := monoNow - time.Duration(trace.StartMonotimeNs)
-	endDelta := monoNow - time.Duration(trace.EndMonotimeNs)
-	goStartDelta := monoNow - time.Duration(trace.GoStartMonotimeNs)
+func (s *HTTPRequestSpan) Inside(parent *HTTPRequestSpan) bool {
+	return s.RequestStart >= parent.RequestStart && s.End <= parent.End
+}
+
+func (s *HTTPRequestSpan) Timings() (time.Time, time.Time, time.Time) {
```

Comment:
for better API readability, I'd maybe use named return values in the signature:

```go
func (s *HTTPRequestSpan) Timings() (goroutineStart time.Time, requestStart time.Time, end time.Time)
```

Or even returning a structure:
```go
type Timings struct {
    GoroutineStart time.Time
    RequestStart time.Time
    End time.Time
}
```

Reply:
Great suggestion! I'll fix it in this PR.